Yoast/yoast-components
# Updated Onboarding Wizard header

## Summary

This PR can be summarized in the following changelog entry:

* Updated onboarding wizard header

## Test instructions

This PR can be tested by following these steps:

* [edit by andrea]: run `grunt build` to compile the CSS
* Run the test environment and see the changes in the wizard.

Fixes #248
